src/Platform/Microsoft.Testing.Platform/Configurations/AggregatedConfiguration.cs

```diff
@@ -52,7 +52,13 @@ public AggregatedConfiguration(IConfigurationProvider[] configurationProviders)
         }
     }
 
-    public /* for testing */ void SetCurrentWorkingDirectory(string workingDirectory)
+    internal /* for testing */ void SetResultDirectory(string resultDirectory)
```

Comment:
`SetResultDirectory` is new, so made it internal by default. If you mean why I moved `SetCurrentWorkingDirectory` to internal, that is because it's only used for testing so I see no purpose in keeping it public for now. Should it be the case in the future to make it public, we can deal with that later. WDYT ?

Reply:
+1, let's keep public

Reply:
Reverted, thanks.